File: Nível 2/2057 - Fuso Horário/main.lua
valor = io.read()
x1, x2, x3 = valor:match("(%S+)%s+(%S+)%s+(%S+)")

x1 = tonumber(x1)
x2 = tonumber(x2)
x3 = tonumber(x3)

calc = x1 + x2 + x3

if 0 <= calc and calc <= 23 then
    print(calc)
elseif calc > 24 then
    calc = calc - 24
    print(calc)
elseif calc < 0 then
    calc = math.abs(calc)
    calc = 24 - calc
    print(calc)
end

File: Nível 2/2057 - Fuso Horário/main.py
valor = input("A: ")
x1,x2,x3 = valor.split(" ")

x1 = int(x1)
x2 = int(x2)
x3 = int(x3)

calc = x1 + x2 + x3 

if 0 <= calc <= 23:
    print(calc)
elif calc > 24:
    calc = calc - 24
    print(calc)
elif calc < 0:
    calc = abs(calc)
    calc = 24 - calc 
    print(calc)